mission/joystick_interface_auv/config/param_joystick_interface_auv.yaml

```diff
@@ -0,0 +1,8 @@
+joystick_interface:
+  ros__parameters:
+    surge_scale_factor:  60.0
+    sway_scale_factor:   60.0
+    yaw_scale_factor:    60.0
+    heave_scale_factor:  35.0
+    roll_scale_factor:  -30.0
```

Comment:
why is this scaling value negative? If it returns the correct output then it should be remapped.